src/ruby/integration/bundler_2_test.go

```diff
@@ -1,22 +0,0 @@
-package integration_test
-
-import (
-	"github.com/cloudfoundry/libbuildpack/cutlass"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("CF Ruby Buildpack", func() {
-	var app *cutlass.App
-
-	AfterEach(func() { app = DestroyApp(app) })
-
-	It("works with old version of bundler 2", func() {
-		app = cutlass.New(Fixtures("bundler_2"))
-		PushAppAndConfirm(app)
-		Expect(app.Stdout.String()).To(MatchRegexp(`Your Gemfile.lock was bundled with bundler 2\.\d+\.\d+, which is incompatible with the current bundler version \(2\.\d+\.\d+\)`))
-		Expect(app.Stdout.String()).To(ContainSubstring(`Deleting "Bundled With" from the Gemfile.lock`))
-		Expect(app.GetBody("/")).To(ContainSubstring("Hello world!"))
```

Comment:
if we support building apps with EOS Bundler versions, this feels like something we should test

Reply:
When this test was written it wasn't really designed to be an EOL test, bundler 2 was just new. The log statements were added later.

Bundler 2 isn't EOL, and I'm not aware of any reason why we should care about testing that apps bundled with older bundlers should work with the buildpack. My understanding is that bundler handles this. It's not a buildpack behavior.

Reply:
It looks like we actually delete the line in the Gemfile.lock file that causes issues here - https://github.com/cloudfoundry/ruby-buildpack/blob/40bdf71ca85ff0865012e464d15cc4f287d0e392/src/ruby/supply/supply.go#L842-L843 and then redo the build. Checking that deleting something from a lockfile leads to a successful build seems important to test. Especially because this. might change in newer bundler versions

Reply:
Ok. I think that's fair feedback. I'd be happy to include that.